### PR TITLE
feat: update vehicle model

### PR DESCRIPTION
**Pull request by emsameen**
## Describe your changes

update vehicle model: 

- change collection api usage
- mark precated classes
- update SDK version

## Issue ticket number and link

AB_#11242

## Checklist - Manual tasks

* [x] Examples are executing successfully
* [ ] Created/updated unit tests. Code Coverage percentage on new code shall be >= 80%.
* [ ] Created/updated integration tests.
* [ ] Devcontainer can be opened successfully
* [ ] Devcontainer can be opened successfully behind a corporate proxy
* [ ] Devcontainer can be re-built successfully
* [ ] Extended the documentation (e.g. README.md, CONTRIBUTING.md, Velocitas)

### CODE REVIEW

File: examples/vdb-queries/src/main.py [hunk 1]
```diff
@@ -85,13 +84,15 @@ async def on_start(self):
         await asyncio.sleep(5)
         await vehicle.Speed.subscribe(on_speed_update)
         (
-            await vehicle.Cabin.Seat.element_at(1, 1)
+            await vehicle.Cabin.Seat.Row(1)
+            .Pos(1)
```
Comment (BjoernAtBosch) [inline examples/vdb-queries/src/main.py:88]: I would prefer one line (but don't really care):
```suggestion
            await vehicle.Cabin.Seat.Row(1).Pos(1)
```

File: examples/vdb-queries/src/main.py [hunk 2]
```diff
@@ -85,13 +84,15 @@ async def on_start(self):
...
             .Position.join(vehicle.Speed)
             .where("Vehicle.Cabin.Seat.Row1.Pos1.Position < 100")
             .subscribe(on_seat_speed_update)
         )
         (
-            await vehicle.Cabin.Seat.element_at(1, 1)
+            await vehicle.Cabin.Seat.Row(1)
+            .Pos(1)
```
Comment (BjoernAtBosch) [inline examples/vdb-queries/src/main.py:95]:
```suggestion
            await vehicle.Cabin.Seat.Row(1).Pos(1)
```

File: requirements-links.txt [hunk 3]
```diff
@@ -1 +1 @@
-git+https://github.com/eclipse-velocitas/vehicle-model-python.git@v0.2.0
+git+https://github.com/eclipse-velocitas/vehicle-model-python.git@v0.3.0-beta
```
Comment (BjoernAtBosch) [inline requirements-links.txt:1]: Don't forget to remove "-beta"

**Review by BjoernAtBosch — APPROVED** [review]

File: requirements-links.txt (hunk 3 above)
Comment (emsameen) [reply requirements-links.txt:1]: sure :)

File: examples/vdb-queries/src/main.py (hunk 1 above)
Comment (emsameen) [reply examples/vdb-queries/src/main.py:88]: I agree :). However, this was split by the pre-commit which I can't change back

File: examples/vdb-queries/src/main.py (hunk 2 above)
Comment (emsameen) [reply examples/vdb-queries/src/main.py:95]: can't have them in one line due to the linter.

**Pull request merged**